programmers/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,24 @@
+function solution(board, moves) {
+    
+    // const newMoves = moves.map((x) =>  x - 1);
+    let count = 0;
+    let stack = [];
+    
+    for (let i = 0; i < moves.length; i++) {
+        let now = moves[i] -1;
```

Comment:
```suggestion
        let crane = moves[i] - 1;
```
띄어쓰기 주의해주시고 변수명은 명사로 지정해주어야 합니다!